sirepo/feature_config.py

```diff
@@ -109,12 +118,12 @@ def b(msg, dev=False):
             'Names of root packages that should be checked for codes and resources. Order is important, the first package with a matching code/resource will be used. sirepo added automatically.',
         ),
         proprietary_sim_types=(set(), set, 'codes that require authorization'),
-        #TODO(robnagler) make this a sim_type config like srw and warpvnd
-        rs4pi_dose_calc=(False, bool, 'run the real dose calculator'),
+        sim_global=dict(
+            hide_guest_warning=b('Hide the guest warning in the UI', dev=True),
```

Comment:
What's the advantage to adding the `sim_global` dict instead of making `hide_guest_warning` at the top-level? Should things like `package_path` be added to this?

Reply:
The sim-specific cfg goes into the app schema (which is where ```sirepo.js``` looks for ```hide_guest_warning```) - my thought was to keep top-level stuff and not add it piecemeal. I guess anything that the client needs to know can go there

Reply:
Ah makes sense. Thanks.